### PR TITLE
Make shuttle airlocks not snapcardinals

### LINKED ISSUE #33478: Shuttle airlocks are snapcardinals
IDK when this broke I haven't looked at one in a minute but the nub is now perpetually south.

### PR DESCRIPTION
**Pull request by Winkarst-cpu**
## About the PR
Title.

## Why / Balance
Fixes #33478.

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Winkarst-cpu** [comment]
Unrelated failed check?

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/shuttle.yml [hunk 1]
```diff
@@ -69,6 +70,7 @@
   components:
   - type: Sprite
     sprite: Structures/Doors/Airlocks/Glass/shuttle.rsi
+    snapCardinals: false
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/shuttle.yml:73]:
```suggestion
```
inherited from AirlockShuttle

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/shuttle.yml [hunk 2]
```diff
@@ -118,3 +121,4 @@
   components:
   - type: Sprite
     sprite: Structures/Doors/Airlocks/Standard/shuttle_syndicate.rsi
+    snapCardinals: false
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/shuttle.yml:124]:
```suggestion
```
inherited

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Review by slarticodefast — APPROVED** [review]
Tested and works as intended.
Thanks for the fix!

**Pull request merged**